packages/e2e/README.md

```diff
@@ -44,14 +44,16 @@ Put the a file named `<package_name>_e2e_test.dart` in the app' `test_driver` di
 ```dart
 import 'dart:async';
 import 'dart:io';
+import 'package:e2e/common.dart' as common;
 import 'package:flutter_driver/flutter_driver.dart';
 
 Future<void> main() async {
   final FlutterDriver driver = await FlutterDriver.connect();
-  final String result =
+  final String jsonResult =
       await driver.requestData(null, timeout: const Duration(minutes: 1));
+  final common.Response response = common.Response.fromJson(jsonResult);
```

Comment:
What do you think about putting the entire main into the e2e package, e.g. `Future<void> main() async => e2e.main();`

It would make the e2e package less flexible but we wouldn't have to worry about it being used in unintended ways, semantic versioning of the `Response` class, etc. It would be easier to add improvements without developers updating their files. For example we might want to print the failures as they occur instead of waiting for everything to finish before reporting failures.

We could still do some driver testing of the failure case, but still hide the `Response` class from clients of the package, but the imports wouldn't need to be part of the documented boilerplate.

I'm not terribly wedded to this idea so if you prefer the current way of doing things let me know and we can stick to that.

Reply:
done! thanks for the suggestion!